tests/settings.py

```diff
@@ -100,3 +100,7 @@
     'root': {
         'handlers': ['console'],
     }}
+
+
+# USE_TZ = True
```

Comment:
Should this be removed?

Reply:
I left it intentionally so someone could test example with timezone. But if it is not clear it should be removed probably.